wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/detail/CertificateDetailFragment.kt

```diff
@@ -847,7 +847,8 @@ class CertificateDetailFragment : Fragment() {
 			} else if (hours <= 72L) {
 				context.getString(R.string.wallet_validity_since_hours_plural).replace("{HOURS}", hours.toString())
 			} else {
-				context.getString(R.string.wallet_validity_since_hours_plural).replace("{HOURS}", "72")
+				//to be grammatically correct the ago is removed for the english case
```

Comment:
Why not use a new strings?

Reply:
Because we didn't want a term on POEditor with only one translation.